### DISCUSSION TITLE
Add access control to languages

### DISCUSSION THREAD

**Discussion by dave-calleja** [payloadcms/payload#1496] [Feature Requests & Ideas]
I am finding payload awesome to work with however where the access rules shine in most parts, they seem totally missing in others. One such case is giving a user access to only certain languages. One use case could be that you want to give access to a translator but only for a certain language.

Is this possible with current features? or Does this need to be added as a feature? 

Thanks!

**Comment by jmikrut** [comment]
Hey @dave-calleja — this is a great thought.

Super easy to add too. Gonna add it to our roadmap. 👍

**Reply by dave-calleja** [reply]
Any news on this?

**Reply by MaeLany** [reply]
¿Alguna novedad? Me encantaría también esa gran idea.

**Reply by dave-calleja** [reply]
👀

**Comment by jacobsfletch** [comment]
You can now add access control to languages by reading from the `req.locale` argument. For example:

```ts
access: {
  create: ({ req }) => req.locale !== 'es'
}
```

**Comment by JarrodMFlesch** [comment]
We also have [filterAvailableLocales](https://payloadcms.com/docs/configuration/localization#filter-available-options) which is more of a ui improvement. It allows you to show specific locales per user.
